content/docs/1.4.0/volumes-and-nodes/storage-tags.md

```diff
@@ -28,10 +28,34 @@ When creating a volume, specify the disk tag and node tag in the UI.
 
 ### Kubernetes
 
-Use Kubernetes StorageClass setting to specify tags.
+Use Kubernetes StorageClass parameters to specify tags.
 
-For example:
+For example, edit the config map named `longhorn-storageclass` and add parameter `nodeSelector: "storage,fast"`. Here is an example snippet:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+data:
+  storageclass.yaml: |
+    kind: StorageClass
+    apiVersion: storage.k8s.io/v1
+    metadata:
+      name: longhorn
+      annotations:
+        storageclass.kubernetes.io/is-default-class: "true"
+    provisioner: driver.longhorn.io
+    allowVolumeExpansion: true
+    reclaimPolicy: "Delete"
+    volumeBindingMode: Immediate
+    parameters:
+      numberOfReplicas: "3"
+      staleReplicaTimeout: "480"
+      diskSelector: "ssd"
+      nodeSelector: "storage,fast"
```

Comment:
Retain the comment at the end of this example:
```
If Longhorn is installed via Helm, you can achieve that by editing `persistence.defaultNodeSelector` in [values.yaml](https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/chart/values.yaml).
```

Reply:
Thanks a lot for your suggestion, I have revised the description. Thank you so much!